_posts/2023-06-16-End-to-end-acknowledgements-in-Data-Prepper.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title:  "End-to-end acknowledgements in Data Prepper"
+authors:
+- kkondaka
+- dlv
+date: 2023-06-16 10:00:00 -0700
+categories:
+  - technical-post
+
+meta_keywords: Data Prepper E2E, Data Prepper sink, Data Prepper pipeline as the sink
+meta_description: Learn how Data Prepper provides data durability through the use of end to end acknowledgements by delivering data to the sink before notifying the source.
+
+---
+
+## The need: Improving durability
```

Comment:
Lets remove this heading. It looks odd on the rendered site and the same idea is conveyed with the line "To solve challenges with data durability, Data Prepper now has capability to use end-to-end acknowledgements."